### PR TITLE
Add `streamlit` and `preprocess`/`postprocess` to docs

### PR DESCRIPTION
**Pull request by aguschin**
Significantly changed pages:
- https://mlem-ai-release-0-4-0-paqi5ijw.herokuapp.com/doc/user-guide/serving/streamlit
- https://mlem-ai-release-0-4-0-paqi5ijw.herokuapp.com/doc/user-guide/models
- https://mlem-ai-release-0-4-0-paqi5ijw.herokuapp.com/doc/api-reference/save

This is for 0.4.0 release of MLEM

### CODE REVIEW

**Review by mike0sv — APPROVED** [review]

**Comment by aguschin** [comment]
@jorgeorpinel, I'm merging this since we're releasing 0.4.0, but will be happy to improve these pages later.

File: scripts/docs/utils.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "MLEM project": "[MLEM project](/doc/user-guide/project-structure)",
     "metafile": "[MLEM Object](/doc/user-guide/basic-concepts)",
     "MLEM configuration": "[MLEM config](/doc/user-guide/configuration)",
-    "MLEM config": "[MLEM config](/doc/user-guide/configuration)",
+    # "MLEM config": "[MLEM config](/doc/user-guide/configuration)",  # repeats itself
```
Comment (jorgeorpinel) [inline scripts/docs/utils.py:13]: Should we just remove that line?

File: content/docs/user-guide/serving/streamlit.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+MLEM can use Streamlit to expose your model to external users with a nice
+auto-generated UI.
+
+## Requirements
+
+```bash
+pip install mlem[streamlit]
+# or
+pip install fastapi uvicorn streamlit streamlit_pydantic
```
Comment (jorgeorpinel) [inline content/docs/user-guide/serving/streamlit.md:15]: Let's try to be consistent now that https://github.com/iterative/mlem.ai/pull/268 is in 🙂 

````suggestion
```cli
$ pip install mlem[streamlit]
# or
$ pip install fastapi uvicorn streamlit streamlit_pydantic
````

Comment (jorgeorpinel) [reply content/docs/user-guide/serving/streamlit.md:15]: p.s. looks like lots of `$` are still missing in other places ([e.g.](https://mlem.ai/doc/user-guide/serving/rabbitmq))... Didn't catch that in #268.

File: content/docs/user-guide/serving/streamlit.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+🖇️  Adding route for /predict_proba
+Checkout openapi docs at <http://0.0.0.0:8080/docs>
+INFO:     Started server process [42223]
+INFO:     Waiting for application startup.
+INFO:     Application startup complete.
+INFO:     Uvicorn running on http://0.0.0.0:8080 (Press CTRL+C to quit)
+
+  You can now view your Streamlit app in your browser.
+
+  URL: http://0.0.0.0:80
```
Comment (jorgeorpinel) [inline content/docs/user-guide/serving/streamlit.md:37]: Remove all output from the sample? We didn't add it in https://mlem.ai/doc/user-guide/serving/fastapi, for example.

File: content/docs/user-guide/serving/streamlit.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+```
+
+Serving a model with Streamlit automatically expose FastAPI endpoint as well.
```
Comment (jorgeorpinel) [inline content/docs/user-guide/serving/streamlit.md:40]: Link to https://mlem.ai/doc/user-guide/serving/fastapi ?

**Review by jorgeorpinel — COMMENTED** [review]
Just a few notes 👇🏼

**Comment by aguschin** [comment]
@jorgeorpinel, did you fixed this in any other PRs or I can go ahead and do that?

**Comment by jorgeorpinel** [comment]
I did not address any of this myself. Thanks

**Pull request merged**